src/cpu/aarch64/xbyak_aarch64/xbyak_aarch64/xbyak_aarch64.h

```diff
@@ -46,7 +46,7 @@
 #include <cmath>
 #include <functional>
 
-#if defined(__GNUC__)
+#if defined(__GNUC__) && !defined(__MINGW64__)
```

Comment:
What are we guarding against here? This is a complicated intersection that may deserve a comment.

Reply:
We can't use stuff like
```
#include <sys/mman.h>
#include <unistd.h>
``` 
In Windows, regardless of the compiler.

Reply:
In which case, should this be 
```
#if defined(__GNUC__) && !defined(_WIN32)
```

Or maybe without the `__GNUC__`? Although a change of this kind would require more testing, so I would understand if you were against it.

Reply:
Yes, I can't test all OSes / compilers. That's why my changes were as orthogonal as possible.